### PR TITLE
Turns holsters into an accessory

### PR DESCRIPTION
**Pull request by Alex-Selvey**
Original code by ike709:
Holsters are now an accessory with viewable storage

Merged code created by ike709 to work with sunset's file structure. Holsters are now accessories and can be put in pockets slots.

## Changelog
:cl: Original code by ike709 ported by MemeProof
tweak: Holsters are now accessories
imageadd: added object for holsters
/:cl:

holster

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -192,7 +192,7 @@
 	new /obj/item/holosign_creator/security(src)
 	new /obj/item/reagent_containers/spray/pepper(src)
 	new /obj/item/clothing/suit/armor/vest/det_suit(src)
-	new /obj/item/storage/belt/holster/full(src)
+	//new /obj/item/storage/belt/holster/full(src) disabled due to change in holsters
```
Comment (ghost) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: should be `sunset - disabled due to change in holsters`

File: code/game/objects/items/storage/belt.dm [hunk 2]
```diff
@@ -533,7 +533,7 @@
 	STR.can_hold = typecacheof(list(
 		/obj/item/ammo_casing/shotgun
 		))
-
+/* Disabled due to change to holsters to be sloted in pockets instead of belt slot
```
Comment (ghost) [inline code/game/objects/items/storage/belt.dm]: `/* Sunset - Disabled due to change to holsters to be sloted in pockets instead of belt slot`

**Review by ghost — CHANGES_REQUESTED** [review]
sorry that i'm anal about this, but I prefer to be able to Ctrl+F for 'sunset' in case of conflict or large changes.

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm (hunk 1 above)
Comment (Alex-Selvey) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: No problem I completely agree and will do comments like this going forward.

File: code/game/objects/items/storage/belt.dm (hunk 2 above)
Comment (Alex-Selvey) [reply code/game/objects/items/storage/belt.dm]: Also updated this as well

**Pull request merged**